packages/@okta/i18n/src/properties/login.properties

```diff
@@ -399,7 +399,7 @@ enroll.yubikey.subtitle = Insert your YubiKey into a USB port and tap it to gene
 # Enroll TOTP
 enroll.totp.title = Setup {0}
 enroll.totp.selectDevice = Select your device type
-enroll.totp.downloadApp = Download <a href="{0}" class="inline-link">{1} from the {2}</a> onto your mobile device.
+enroll.totp.downloadApp = Download <a href="{0}" target="_blank" rel=“noreferer noopener” class="inline-link">{1} from the {2}</a> onto your mobile device.
```

Comment:
@magizh-okta Seeing the change in the context of the codebase makes me realize the nuance of what you were asking in the SIW OH. This HTML is embedded _in_ the properties file and rendered without escaping [`{{{appStoreLinkText}}}`](https://github.com/okta/okta-signin-widget/blob/master/src/EnrollTotpController.js#L31). This makes it particularly dangerous and fragile since the translations are provided by an outside vendor.

I've faced a similar situation before, and the solution was to choose a notation (a subset of markdown) that can safely turn the string into a link in a controlled and tested way. There are other cases of embedding a link in the properties file, but I'd argue it's something we should fix.

Thought this was funny: [`≤å ĥŕéƒ≂″ 한Ӝฐโ{0}″ çļåšš≂″îñļîñé‐ļîñķ″≥`](https://github.com/okta/okta-signin-widget/blob/master/packages/@okta/i18n/src/properties/login_ok_PL.properties#L345) from the pseudo-loc properties file.

Reply:
@lesterchoi-okta yeah that makes sense. the alternate approach that we do in admin UI is not recommended for now https://okta.slack.com/archives/CAZH2MWEL/p1628113991046900?thread_ts=1628113116.044800&cid=CAZH2MWEL